[Logging Service: 12]
[2024-11-05 17:11:47.831 -0300] [INFO] [cadence.virtuoso.ade.loggingservice] - HistoryRecorderManager init {"dbBasePath": "/home/cinovador/Documents/course_files/digital_logic/practice_2/logic_gates/lab3_nor/maestro/results/maestro"}
[2024-11-05 17:11:47.832 -0300] [INFO] [cadence.virtuoso.ade.loggingservice] - useGRPC grpc service... {"brokerAddress": "cinova05.lesc.ufc.br:40141", "serviceName": "DaVinciService_14928_1730827588", "axlSession": "fnxSession2"}
[2024-11-05 17:11:47.832 -0300] [PROCESS] [cadence.virtuoso.ade.loggingservice] - Logging Service has successfully started. Waiting for incoming messages ... {"hostname": "cinova05.lesc.ufc.br", "processid": 96361, "logfile": "/home/cinovador/Documents/course_files/digital_logic/practice_2/logs_cinovador/logs0/LoggingService12.log"}
[2024-11-05 17:11:48.097 -0300] [INFO] [cadence.virtuoso.ade.loggingservice] - logging service gRPC started {"broker address": "cinova05.lesc.ufc.br:35467"}
[2024-11-05 17:19:56.365 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "UNIQUE constraint failed: location.point, location.test, location.tool", "sql": "INSERT INTO location (tool, file, point, test, line) VALUES ('unknown', '', '', '', '0');"}
[2024-11-05 17:19:56.365 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "delete from meta;"}
[2024-11-05 17:19:56.365 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "insert into meta (key, value) values('HistorySimDir','/home/cinovador/Documents/course_files/digital_logic/practice_2/logic_gates/lab3_nor/maestro/logic_gates/lab3_nor/maestro/results/maestro/ExplorerRun.0');"}
[2024-11-05 17:19:56.365 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "insert into meta (key, value) values('WorkDir','/home/cinovador/Documents/course_files/digital_logic/practice_2');"}
[2024-11-05 17:19:56.365 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "insert into meta (key, value) values('version','1');"}
[2024-11-05 17:19:56.365 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='exprOutputs.log.14.0.1' WHERE tool='Expression Evaluator' AND file='/home/cinovador/Documents/course_files/digital_logic/practice_2/logic_gates/lab3_nor/maestro/logic_gates/lab3_nor/maestro/results/maestro/ExplorerRun.0/exprOutputs.log.14.0.1';"}
[2024-11-05 17:19:56.365 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='logs_cinovador/logs0/Job14.log' WHERE tool='Simulation Monitor' AND file='/home/cinovador/Documents/course_files/digital_logic/practice_2/logs_cinovador/logs0/Job14.log';"}
[2024-11-05 17:19:56.365 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='1/logic_gates_lab3_nor_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cinovador/Documents/course_files/digital_logic/practice_2/logic_gates/lab3_nor/maestro/logic_gates/lab3_nor/maestro/results/maestro/ExplorerRun.0/1/logic_gates_lab3_nor_1/psf/spectre.out';"}
[2024-11-05 17:19:56.365 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:19:56.365 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:19:56.912 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert location failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:19:56.912 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:14.239 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert location failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:17.280 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:17.281 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:17.281 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:17.281 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:17.281 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:17.281 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert location failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:17.381 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert location failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:18.329 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:18.330 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:18.365 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:20.648 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert location failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:20.649 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.366 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.367 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.367 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.367 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.367 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.368 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.368 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.368 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.368 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.369 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.369 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.369 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.369 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.369 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.369 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.369 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.868 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.868 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.868 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.868 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.869 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.869 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.869 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.869 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.870 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.870 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.870 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.870 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.870 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:21.962 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert location failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:22.161 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:22.379 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:22.869 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:20:23.312 -0300] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-05 17:31:42.240 -0300] [INFO] [cadence.virtuoso.ade.loggingservice] - Got exit signal in processControlMessage. Will exit. {}
[2024-11-05 17:31:43.243 -0300] [PROCESS] [cadence.virtuoso.ade.loggingservice] - Current memory used: 91.1094 MBytes, peak memory used: 91.1523 MBytes {"hostname": "cinova05.lesc.ufc.br", "processid": 96361, "logfile": "/home/cinovador/Documents/course_files/digital_logic/practice_2/logs_cinovador/logs0/LoggingService12.log"}
[2024-11-05 17:31:43.243 -0300] [PROCESS] [cadence.virtuoso.ade.loggingservice] - Stopping Logging Service process. {"hostname": "cinova05.lesc.ufc.br", "processid": 96361, "logfile": "/home/cinovador/Documents/course_files/digital_logic/practice_2/logs_cinovador/logs0/LoggingService12.log"}
[2024-11-05 17:31:43.243 -0300] [PROCESS] [cadence.virtuoso.ade.loggingservice] - Logging Service stop!!!! {"hostname": "cinova05.lesc.ufc.br", "processid": 96361, "logfile": "/home/cinovador/Documents/course_files/digital_logic/practice_2/logs_cinovador/logs0/LoggingService12.log"}
